Sprint-1/fix/median.js

```diff
@@ -1,11 +1,14 @@
-// Fix this implementation
-// Start by running the tests for this function
-// If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
-
-function calculateMedian(list) {
-  const middleIndex = Math.floor(list.length / 2);
-  const median = list.splice(middleIndex, 1)[0];
-  return median;
-}
-
-module.exports = calculateMedian;
+// Fix this implementation
+// Start by running the tests for this function
+// If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
+
+function calculateMedian(list) {
+  list.sort((a, b) => a - b);
+  const middleIndex = Math.floor(list.length / 2);

```

Comment:
It is nice to see you use sort method and arrow functions inside of your code. You can also  write your own  logic as well, consider this one as a challange